<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Anndy-PC" name="cn.AnndyTsal.PO.PO_simplify.Case.HomeTestCase1" tests="1" failures="0" timestamp="06 八月 2019 10:09:44 GMT" time="62.568" errors="1">
  <testcase name="loginCase" time="62.568" classname="cn.AnndyTsal.PO.PO_simplify.Case.HomeTestCase1">
    <error type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
at message : 预期为空 实际不为空 expected [true] but found [false] in method : .loginCase(HomeTestCase1:39)
at cn.AnndyTsal.PO.PO_simplify.Case.HomeTestCase1.loginCase(HomeTestCase1.java:39)
]]>
    </error>
  </testcase> <!-- loginCase -->
</testsuite> <!-- cn.AnndyTsal.PO.PO_simplify.Case.HomeTestCase1 -->
